### ISSUE TITLE
Airmon-ng does not recognize ipw2200 1.2.0

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#28]
**Reported by latinsud on 23 Dec 2006 19:29 UTC**

Airmon-ng has a list of invalid ipw2200 driver versions. This list is not currently correct.

Current code is:
```
if { echo "$MODINFO" | grep -E '^1\.0\.(0|1|2|3)$|^1\.[; }
...
if { echo "$MODINFO" | grep -E '^1\.0\.(5|7|8|11)$|^1\.[1-9](1-9]')' ; }
```

This blacklists current working version, 1.2.0, as well as future versions. I don't remember the whole versioning history of the driver, but iirc it'd be safe to assume that version 1.1.1 and higher worked fine. So i suggest:
```
if { echo "$MODINFO" | grep -E '^1\.0\.(0|1|2|3)$' ; }
...
if { echo "$MODINFO" | grep -E '^1\.0\.(5|7|8|11)$' ; }
```

There's also an interesting feature of driver, starting from version 1.1.2, that appends a list of capabilities to version, like: 1.2.0dmprq. The 'm' indicates that support for monitor mode was compiled in.

Previously reported in forum
http://tinyshell.be/aircrackng/forum/index.php?topic=685.0

**Comment by aircrack-ng** [comment]
**Comment by misterx on 23 Dec 2006 19:29 UTC**

(In #39) Fixed: airmon-ng does not recognize ipw2200 1.2.0 (Closes: #28)

**Comment by aircrack-ng** [comment]
**Comment by the prophet on 23 Dec 2006 19:29 UTC**

Seems to be not fixed, or another problem?

airmon-ng start eth1 11

usage: airmon-ng <start|stop> <interface> [channel]

Interface       Chipset         Driver

eth1            Centrino b/g    '''ipw2200/usr/sbin/airmon-ng: line 111: startStdIface: command not found'''

ath9            Atheros         madwifi-ng VAP (parent: wifi0)

**Comment by aircrack-ng** [comment]
**Comment by the prophet on 23 Dec 2006 19:29 UTC**

Ok i found the Problem, 
Line 15 fstartStdIface()!=  Line 111 startStdIface()

Correct Line 15 to startStdIface() and it works perfectly...

**Comment by aircrack-ng** [comment]
**Comment by misterx on 23 Dec 2006 19:29 UTC**

(In #49) Fixed a stupid bug described in ticket 28 (Closes: #28).
Only show usage when not all parameters are given (or first parameter is not start/stop).
Show an error when trying to stop wifiX (madwifi-ng).
